src/layout/AccountTeaser.tsx

```diff
@@ -29,10 +28,9 @@ function AccountTeaser(props: PropTypes) {
   const [qrVisible, setQrVisible] = useState(false);
   const {theme} = useTheme();
   const {accounts} = useAccounts();
-  const {currentNetwork} = useContext(NetworkContext);
-  const {api} = useContext(ChainApiContext);
   const account = accounts[0]; // TODO: change this when adding multi account support
```

Comment:
are we still using this?

Reply:
The file is actually not used anywhere, I just kept it because thought maybe it'll be used in future. should I remove it entirly?

Reply:
I see.. Let's create an issue to remove it later if won't use it (I think at some point we need to revisit the accounts list)